ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/aaa.md

```diff
@@ -1,23 +1,47 @@
 # aaa
 # Table of Contents
 
+- [aaa](#aaa)
```

Comment:
Not suggesting to change anything here, as I'd rather create a dedicated PR for that. But to brainstorm some TOC related ideas:
- IMO, if `aaa` is lvl1 header, everything else must be lvl2 or deeper
- I'd consider enforcing a specific Markdown TOC plugin for contributors. For example `yzhang.markdown-all-in-one`.  @ClausHolbechArista